### ISSUE TITLE
BedrockChat throws error on tool use with "anthropic.claude-3-5-sonnet-20240620-v1:0" and "anthropic.claude-3-haiku-20240307-v1:0"

### ISSUE THREAD

**Issue by goldeneagle3636** [langchain-ai/langchainjs#7237]
### Checked other resources

- [X] I added a very descriptive title to this issue.
- [X] I searched the LangChain.js documentation with the integrated search.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I am sure that this is a bug in LangChain.js rather than my code.
- [X] The bug is not resolved by updating to the latest stable version of LangChain (or the specific integration package).

### Example Code

```typescript
import { BedrockChat } from "@langchain/community/chat_models/bedrock"
import { tool } from "@langchain/core/tools"
import { AgentExecutor, createToolCallingAgent } from 'langchain/agents'
import { ChatPromptTemplate } from '@langchain/core/prompts'

// anthropic.claude-3-haiku-20240307-v1:0
const model = new BedrockChat({
  model: "anthropic.claude-3-5-sonnet-20240620-v1:0",
  region: "us-east-1",
  credentials: {
    accessKeyId: <>,
    secretAccessKey: <>,
  }
})

const imageTool = tool(
  async () => {
    return "image url";
  },
  {
    name: "Get-Image-Tool",
    description:
      "Use this tool if the user asks you to send them an image/picture",
  }
);


const systemMessage = "you are a helpful assistant"

const prompt = ChatPromptTemplate.fromMessages([
  ['system', systemMessage],
  ['placeholder', '{chat_history}'],
  ['human', '{input}'],
  ['placeholder', '{agent_scratchpad}'],
])

const agent = createToolCallingAgent({
  llm: model,
  tools: [imageTool],
  prompt,
  streamRunnable: true,
})
const agentExecutor = new AgentExecutor({
  agent,
  tools: [imageTool],
  verbose: false,
})

console.log("Agent created")

const result = await agentExecutor.invoke({input:  "use the image tool to get the image url for a picture of a falcon and report what you get back"})

console.log(result)
```

### Error Message and Stack Trace (if applicable)

file:///.../node_test/node_modules/@langchain/community/dist/utils/bedrock/anthropic.js:131
                throw new Error("Unsupported message content format");
                      ^

Error: Unsupported message content format
    at file:///.../c/node_test/node_modules/@langchain/community/dist/utils/bedrock/anthropic.js:131:23
    at Array.map (<anonymous>)
    at _formatContent (file:///.../c/node_test/node_modules/@langchain/community/dist/utils/bedrock/anthropic.js:101:39)
    at file:///.../c/node_test/node_modules/@langchain/community/dist/utils/bedrock/anthropic.js:190:30
    at Array.map (<anonymous>)
    at formatMessagesForAnthropic (file:///.../c/node_test/node_modules/@langchain/community/dist/utils/bedrock/anthropic.js:147:52)
    at BedrockLLMInputOutputAdapter.prepareMessagesInput (file:///.../c/node_test/node_modules/@langchain/community/dist/utils/bedrock/index.js:131:61)
    at BedrockChat._signedFetch (file:///.../c/node_test/node_modules/@langchain/community/dist/chat_models/bedrock/web.js:688:44)
    at BedrockChat._streamResponseChunks (file:///.../c/node_test/node_modules/@langchain/community/dist/chat_models/bedrock/web.js:737:37)
    at _streamResponseChunks.next (<anonymous>)

Node.js v20.16.0

### Description

I am using the langchain library as illustrated above. It appears that the response from bedrock has a new json format that the BedrockChat class cannot yet support. It seems like the patch is not too in depth but am hoping the langchain team can prioritize making this class work with bedrock again as amazon has changed the output formats.

### System Info

npm --version
10.8.1
node --version
v20.16.0
Platform
MacOS Sonoma 14.4.1 (23E224)

**Comment by jacoblee93** [comment]
CC @bracesproul could you have a look

**Comment by bracesproul** [comment]
Hey @goldeneagle3636 I pushed a fix to PR #7244, however we suggest you start to migrate off of bedrock inside `@langchain/community`, and instead start using the more stable `@langchain/aws` package. Simply replace your chat model instantiation with this:
```typescript
import { ChatBedrockConverse } from '@langchain/aws'

const model = new ChatBedrockConverse({
  model: "anthropic.claude-3-5-sonnet-20240620-v1:0",
  region: "us-east-1",
  credentials: {
    accessKeyId: process.env.BEDROCK_AWS_ACCESS_KEY_ID!,
    secretAccessKey: process.env.BEDROCK_AWS_SECRET_ACCESS_KEY!,
  },
});
```

And you're done!
